render.py

```diff
@@ -23,8 +23,9 @@
 
     with open(config_file) as f:
         params = yaml.load(f, Loader=yaml.SafeLoader)
-        params.setdefault('docker_upload', "dockerhub;ghcr;quay")
-        params.setdefault('zenodo_upload', "yes")
+    params.setdefault('docker_upload', "dockerhub;ghcr;quay")
+    params.setdefault('zenodo_upload', "yes")
+    params.pop('zenodo_metadata')
```

Comment:
A helpful comment message would be useful for new maintainers/or future us in 6+ months.
```suggestion
    params.pop('zenodo_metadata')  # removing the parameters that are not needed for rendering
```